stgit/commands/rebase.py

```diff
@@ -242,13 +256,28 @@ def func(parser, options, args):
     else:
         target = stack.base
 
+    if options.autostash and not iw.worktree_clean():
+        Run('git', 'stash').run()
```

Comment:
We should prefer using `repository.run()` instead of `Run()`. The former sets the `GIT_DIR` environment variable to give Git commands a normalized environment.

Also, I'd also prefer if we used the more explicit `git stash push` here.